src/utils/sendModerationDm.ts

```diff
@@ -12,34 +13,47 @@ import { customSubstring } from "./customSubstring";
  * @param {BeccaLyria} Becca Becca's Discord instance.
  * @param {ModerationActions} action The moderation action taken.
  * @param {User} user The Discord user being moderated.
- * @param {string} guildName The name of the guild the moderation occurred in.
+ * @param {string} guild The guild the moderation occurred in.
  * @param {string} reason The reason for the moderation action.
  * @returns {boolean} True if the message was sent, false otherwise.
  */
 export const sendModerationDm = async (
   Becca: BeccaLyria,
   action: ModerationActions,
   user: User,
-  guildName: string,
+  guild: Guild,
   reason: string
 ): Promise<boolean> => {
   try {
     const embed = new MessageEmbed();
     embed.setTitle(`${action} Notification!`);
     embed.setDescription(
-      `You have received a ${action} in ${guildName} for: \n\n${customSubstring(
-        reason,
-        2000
-      )}`
+      `You have received a ${action} in ${
+        guild.name
+      } for: \n\n${customSubstring(reason, 2000)}`
     );
 
+    if (action === "ban") {
+      const serverID = guild.id;
+      const server = (await ServerModel.findOne({ serverID })) || null;
+
+      if (server) {
+        if (server.appeal_link.length > 0) {
+          embed.addField(
+            "You can appeal your ban using this link: ",
+            server.appeal_link
+          );
+        }
+      }
```

Comment:
```suggestion
      if (server && server.appeal_link.length) {
        embed.addField(
          "You can appeal your ban using this link: ",
          server.appeal_link
        );
      }
```
A few notes here:
- We can run both checks in the single `if` condition so we don't have to nest things.
- `appeal_link` is a string, which means we can assert for truthiness as the only time it would be falsy is if it's an empty string.